Source/WebCore/svg/graphics/filters/SVGFilterGraph.h

```diff
@@ -77,17 +78,19 @@ class SVGFilterGraph {
 
     RefPtr<NodeType> getNamedNode(const AtomString& id) const
     {
-        if (id.isEmpty()) {
-            if (m_lastNode)
-                return m_lastNode;
+        if (!id.isEmpty()) {
+            if (auto sourceNodes = m_sourceNodes.get(id))
+                return sourceNodes;
 
-            return sourceGraphic();
+            if (auto namedNodes = m_namedNodes.get(id))
```

Comment:
And this variable should be named `namedNode` or `node`.

Reply:
Opted for `sourceNode` and `namedNode` and pushed newer commit. :-)